Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Lib/DriverScore.hs

```diff
@@ -191,6 +191,10 @@ createDriverStat currency distanceUnit driverId = do
             totalCoinsConvertedCash = 0.0,
             currency,
             distanceUnit,
+            rating = Nothing,
+            totalRatings = Just 0,
```

Comment:
Why did we populate here with 0, it can be nothing

Reply:
those are inital driverstats, so it should be 0 and i am using this logic for in-code backfilling, where only older driver entries which are Nothing, will be refilled rest will be same